mirrulations-client/src/mirrclient/client.py

```diff
@@ -297,7 +369,7 @@ def job_operation(self):
         """
         job_id, job_url, job_type = self.get_job()
         if job_type == 'attachments':
-            result = perform_attachment_job(job_url)
+            result = self.perform_attachment_job(job_url)
         else:
             result = self.perform_job(job_url)
         self.send_job(job_id, result)
```

Comment:
Would it be possible to also send the job type with this? It would simplify things on the work server side. Otherwise is there an easy way to check that result contains the list of files instead of usual data?

Reply:
or add job_type as another key in the json that gets sent to the work server when using the put results endpoint like this:
{'job_id': job_id, 
'results': job_result,' 
job_type': job_type}

Reply:
I have just implemented that, now the json should contain 'job_id'